torchvision/transforms/transforms.py

```diff
@@ -1385,13 +1393,18 @@ def __init__(self, p=0.5, scale=(0.02, 0.33), ratio=(0.3, 3.3), value=0, inplace
         self.inplace = inplace
 
     @staticmethod
-    def get_params(img, scale, ratio, value=0):
+    def get_params(
+            img: Tensor, scale: Tuple[float, float], ratio: Tuple[float, float], value: Optional[List[float]] = None
```

Comment:
This actually changes the default behavior of the function, as before `value=0` and now `value=None` (going to the string codepath).

I think it's ok as the behavior of `__init__` is unchanged, but if users were calling `get_params` explicitly this would be a BC-breaking change.

Can you rename the PR to include [BC-breaking] in the title?